src/System.Runtime/tests/System.Runtime.Tests.builds

```diff
@@ -2,7 +2,12 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
-    <Project Include="System.Runtime.Tests.csproj"/>
+    <Project Include="System.Runtime.Tests.csproj">
+      <TargetGroup>netstandard1.5</TargetGroup>
+    </Project>
+    <Project Include="System.Runtime.Tests.csproj">
```

Comment:
actually I believe you are right, I didn't change the default here, let I'll go and fix that now.